content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -73,21 +73,19 @@ knows how to convert between them in both directions.  Additionally, any new
 field added in v2 must be able to round-trip to v1 and back, which means v1
 might have to add an equivalent field or represent it as an annotation.
 
-**Rule #3: An API version in a given track may not be deprecated until a new
-API version at least as stable is released.**
+**Rule #3: An API version in a given track may not be deprecated in favor of a less stable API version.**
 
-GA API versions can replace GA API versions as well as beta and alpha API
-versions.  Beta API versions *may not* replace GA API versions.
+  * GA API versions can replace beta and alpha API versions.
+  * Beta API versions can replace earlier beta and alpha API versions, but *may not* replace GA API versions.
+  * Alpha API versions can replace earlier alpha API versions, but *may not* replace GA or beta API versions.
 
-**Rule #4a: Other than the most recent API versions in each track, older API
-versions must be supported after their announced deprecation for a duration of
-no less than:**
+**Rule #4a: minimum API lifetime is determined by the API stability level**
```

Comment:
Do we want to update it similarly for metrics?
Removal to "Stable" metrics would also be surprise for users...

Reply:
that sounds like it makes sense, but I'd like the instrumentation leads to propose that change (in a separate PR)

Reply:
cc @brancz @dashpole ^^

Reply:
Stable metrics may not be removed.

KEP: https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/kubernetes-control-plane-metrics-stability.md#stability-classes
Website Docs: https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/#metric-lifecycle

Reply:
@logicalhan which is correct?

Reply:
the latter..

Reply:
> Stable metrics may not be removed.
> 
> KEP: https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/kubernetes-control-plane-metrics-stability.md#stability-classes Website Docs: https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/#metric-lifecycle

Stable metrics cannot be removed, but they can become deprecated and then removed.

Reply:
> That... doesn't seem very stable :)

+1

[Followup PR sounds fine to me of course - those are tightly coupled.]

Reply:
> > Stable metrics cannot be removed, but they can become deprecated and then removed.
> 
> That... doesn't seem very stable :)

You talked me into 4 releases (it was proposed to be 3 initially). Forever is a much bigger commitment...